libnopegl/src/math_utils.c

```diff
@@ -524,6 +525,18 @@ void ngli_mat4_skew(float * restrict dst, float x, float y, float z, const float
     dst[15] = 1.f;
 }
 
+void ngli_mat4_abs(float *dst, const float *m)
```

Comment:
Not necessarily opposed but do you really have a use case for this? 🤔

I can see it for data vectors, but matrices are like functions/coefficient, so I'm curious about your use case

Reply:
It can be useful if you want to decompose a transformation matrix using certain techniques. I dropped the commit for now.